templates/user/profile.tmpl

```diff
@@ -100,12 +100,12 @@
 					</div>
 				{{else if eq .TabName "stars"}}
 					<div class="stars">
-						{{template "explore/search" .}}
+						{{template "explore/search_repo" .}}
```

Comment:
why not rename it `repo_search` since there was a `repo_list`?

Reply:
ok. I renamed it